Lombiq.BaseTheme.Samples/Manifest.cs

```diff
@@ -7,7 +7,7 @@
 [assembly: Theme(
     Name = "Lombiq Base Theme - Samples",
     Author = "Lombiq Technologies",
-    Version = "1.0.0",
+    Version = "1.0.2-alpha",
```

Comment:
Next time don't skip a minor version when publishing an alpha (the current version at the time was 1.0.0 so this should have been 1.0.1-alpha): https://lombiq.atlassian.net/wiki/spaces/DEV/pages/786857987/Managing+releases+of+open-source+projects#:~:text=When%20publishing%20a%20pre%2Drelease%20version%20for%20something%20temporary%20you%20do%20under%20an%20issue%2C%20then%20always%20use%20the%20next%20patch%20version%20with%20a%20suffix.%20E.g.%20if%20the%20current%20latest%20released%20version%20if%201.2.3%20then%20publish%201.2.4%2Dalpha.0.

Reply:
When I made this, there was a 1.0.1-alpha already on nuget, and it was @DAud-IcI version

![image](https://user-images.githubusercontent.com/15349881/170526837-3e402aa5-df3f-4538-b503-e8f11e53dce2.png)

Reply:
I'll clarify the docs but it refers to the latest **released** version. (All this is important for proper releases on GitHub.)

Reply:
Perhaps it would be a good idea to include the issue code in the version code, such as `1.0.1-alpha.osoe-93.1` or something along the lines, so it won't be confusing when two branches need to make independent pre-releases. What do you think @Piedone ?

Reply:
That did occur to me but it didn't seem like a polite thing to do on NuGet, to mix in our issues. But I suppose these pre-releases are adding some pollution anyway, we can try.